packages/muelu/adapters/stratimikos/Thyra_MueLuTpetraQ2Q1PreconditionerFactory_def.hpp

```diff
@@ -36,8 +36,8 @@
 
 #include "MueLu.hpp"
 
-#include "../../research/q2q1/MueLu_Q2Q1PFactory.hpp"
-#include "../../research/q2q1/MueLu_Q2Q1uPFactory.hpp"
+#include "MueLu_Q2Q1PFactory.hpp"
+#include "MueLu_Q2Q1uPFactory.hpp"
```

Comment:
I am guessing that the includes of these files are not enabled in standard Trilinos builds?  (Otherwise, this would not have compiled before.)  Are there no Trilinos PR builds that enable `HAVE_MUELU_EXPERIMENTAL`?

Reply:
The MueLu team has nightly builds with EXPERIMENTAL=ON. It did compile, but setting the include path correctly helped avoid issues when wrapping using binder.